### PR TITLE
#167484579 adding start script and Procfile for heroku

### PR DESCRIPTION
**Pull request by raymond42**
**What does this PR do?**
Deploys base app on Heroku and configures app.json file to generate review apps on PRs

**Description of Task to be completed?**
add start script
add procfile
Create staging and production apps on Heroku
Create a pipeline for both apps
Enable review apps
Configure app.json file to use Postgres addon

**How should this be manually tested?**
New review apps will be created for each pull request after this.

**Any background context you want to provide?**
> Heroku URLs
  >- https://codinggeeks-ah-backnd-staging.herokuapp.com/
  >- https://codinggeeks-ah-bend-production.herokuapp.com/

**What are the relevant pivotal tracker stories?**
#167484579

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,13 @@
   "main": "index.js",
   "scripts": {
     "build": "rm -rf build && babel -d ./build ./src -s",
+    "start": "node ./build/index.js",
```
Comment (mkiterian) [inline package.json]: we need to first run the build then run the server i.e `"npm run build && node ./build/index.js"`

**Review by mkiterian — CHANGES_REQUESTED** [review]
also `morgan` should be moved to dependencies not just a dev dependency

File: package.json (hunk 1 above)
Comment (raymond42) [reply package.json]: done

**Review by mkiterian — DISMISSED** [review]
Looks good

**Review by mkiterian — APPROVED** [review]
Looks good

**Pull request merged**